src/aparna/stringExample1/AddNumbersInString.java

```diff
@@ -0,0 +1,38 @@
+/*/*Assignment - 25 : 18th April'2021
+ * Program 3:  Write a method which will return sum of each all numbers in each String. Main method should print the output.
+Input : {"Tech1No2Credits3", "Ha6rs8h", "Ra9dh3a", "Sha1sha5nk9","Kan3gan4", "1Sandesh3"}
+Output : 6 14 12 15 7 4
+ * 
+ */
+package aparna.stringExample1;
+
+public class AddNumbersInString {
+
+	int[] sumDigitsInString(String[] inputNames) {
+
+		int[] outputArray = new int[inputNames.length];
+		for (int index = 0; index < inputNames.length; index++) {
+			String target = inputNames[index];
+			int sum = 0;
+			for (int innerIndex = 0; innerIndex < target.length(); innerIndex++) {
+				if (target.charAt(innerIndex) >= 48 && target.charAt(innerIndex) <= 57) {
+					sum = sum + target.charAt(innerIndex) - 48;
+					outputArray[index] = sum;
```

Comment:
for outputArray, you should take counter variable, as you know setting up outputArray on based of outer loop, mean jha pr condition safisfied nhi hogi o skip hoga mean 0 rahegi uski value